drivers/ethernet/eth_stm32_hal.c

```diff
@@ -436,19 +637,50 @@ static int eth_initialize(struct device *dev)
 			K_PRIO_COOP(CONFIG_ETH_STM32_HAL_RX_THREAD_PRIO),
 			0, K_NO_WAIT);
 
+#if defined(CONFIG_SOC_SERIES_STM32H7X)
+    for(uint32_t idx = 0; idx < ETH_RX_DESC_CNT; idx ++) {
```

Comment:
indentation
also you should have a space between "for" and (

Reply:
done